docs/modules/reference/pages/daemons/daemon-config-files/notifd.adoc

```diff
@@ -44,7 +44,17 @@ Generates xref:operation:deep-dive/notifications/introduction.adoc[notifications
 | no
 
 | javamail-configuration.properties
-| Configuration to send notification mails via specific mail servers.
+| Configuration to send notification mails via specific mail servers. Metadata expression can be used for the authenticateUser and authenticatePassword fields in order to reference credentials stored in the secure credentials vault.
```

Comment:
```suggestion
| Configuration to send notification emails via specific mail servers. Metadata expression can be used for the authenticateUser and authenticatePassword fields in order to reference credentials stored in the secure credentials vault.
```

Reply:
Also should have carriage-returns between sentences, right? That's one style thing I _do_ know. :D